config/logstash.yml

```
@@ -345,6 +346,7 @@
 #xpack.management.elasticsearch.cloud_auth: logstash_admin_user:password
 # another authentication alternative is to use an Elasticsearch API key
 #xpack.management.elasticsearch.api_key: "id:api_key"
+#xpack.management.elasticsearch.ssl.ca_trusted_fingerprint: xxxxxxxxxx
 #xpack.management.elasticsearch.ssl.certificate_authority: [ "/path/to/ca.crt" ]
```

```suggestion
#xpack.management.elasticsearch.ssl.certificate_authority: "/path/to/ca.crt"
```
Can we please change this as well? We are expecting a path in the codebase but it is an array in config. I have tested that it fails if we do not remove branckets.
```
Your settings are invalid. Reason: Setting "xpack.management.elasticsearch.ssl.certificate_authority" must be a String. Received: ["/Users/mashhur/Dev/elastic/install/elasticsearch-8.2.1/config/certs/http_ca.crt"] (Array)
```